packages/color-scale-block/src/helpers.tsx

```diff
@@ -0,0 +1,166 @@
+import { ColorProps, ColorScaleBlockRef } from './types';
```

Comment:
Create a `/helpers` folder and a separate file for each of the helper functions and an `index.ts` which exports all of them. You can check out some of the other blocks (like checklist-block).

Don't forget the copyright.